composer/src/main/kotlin/com/gojuno/composer/Main.kt

```diff
@@ -134,21 +178,75 @@ fun main(rawArgs: Array<String>) {
             }
             .toBlocking()
             .first()
+}
 
-    val duration = (System.nanoTime() - startTime)
-
-    val totalPassed = suites.sumBy { it.passedCount }
-    val totalFailed = suites.sumBy { it.failedCount }
-    val totalIgnored = suites.sumBy { it.ignoredCount }
+fun AdbDevice.getTestRunnerFromTestApk(): Single<String> =
```

Comment:
wuuut, why do we need to query device for that?

Can't we parse it from apk? This info should be in the manifest

Reply:
Well, mostly, it seemed to be the simpler way. 
However, I've just found that aapt is capable of parsing the whole manifest as well, so we won't need to do it manually. 
Will change it then.

PS. Actually `getTestRunnerFromTestApk` is duplicating `getTestRunnerForInstalledTestApk` defined below, its just and old one.